soroban-settings/phase1.json

```diff
@@ -301,7 +301,7 @@
           "temp_rent_rate_denominator": 5356800,
           "max_entries_to_archive": 100,
           "bucket_list_size_window_sample_size": 30,
-          "eviction_scan_size": 100000,
+          "eviction_scan_size": 1000000,
```

Comment:
Should we set it to 1024 * 1024 here?

Reply:
I'm not sure that this would work, I think we can only use constants here and not expressions because it's a JSON file.

Reply:
> I'm not sure that this would work, I think we can only use constants here and not expressions because it's a JSON file.

I meant the result of the multiplication, 1048576, for keeping it accurately at 1 MB.